misc/services/mobile-webui/mobile-webui-frontend/src/routes/workflow.js

```diff
@@ -1,20 +1,14 @@
 import WFProcessScreen from '../containers/wfProcessScreen/WFProcessScreen';
 import { push } from 'connected-react-router';
-import { pushHeaderEntry } from '../actions/HeaderActions';
+
+import { getLocation } from '../utils';
 
 export const gotoWFProcessScreen = ({ wfProcess }) => {
   return (dispatch) => {
-    const location = workflowLocation({ workflowId: wfProcess.id });
+    const location = getLocation({ wfProcessId: wfProcess.id });
```

Comment:
i would prefer to have a `getWFProcessScreenLocation` function here in this file.